ext/session/session.c

```diff
@@ -3211,6 +3211,7 @@ static int php_session_rfc1867_callback(unsigned int event, void *event_data, vo
 				if (PS(rfc1867_cleanup)) {
 					php_session_rfc1867_cleanup(progress);
 				} else {
+					SEPARATE_ARRAY(&progress->data);
```

Comment:
I guess Dmitry meant we could fix this in a separate commit before we land the actual COW changes.

fixing it after the COW changes doesnt make much sense though.